src/components/NewRuntimeModal.js

```diff
@@ -306,7 +305,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
       await Ajax().Runtimes.runtime(namespace, Utils.generateRuntimeName()).create({
         runtimeConfig,
         toolDockerImage: newRuntime.toolDockerImage,
-        labels: { saturnIsProjectSpecific: `${selectedLeoImage === PROJECT_SPECIFIC_MODE}` },
+        labels: {},
```

Comment:
I don't think you need to keep `labels` here at all anymore

Reply:
is labels an optional parameter here?

Reply:
Yes, and here's how you can find out: If you take a look at the source of the `create` function, it passes this whole object to `_.merge`. That does a recursive deep merge, so it will combine these labels with the default ones specified there. Since we no longer have any additional labels to add, we can omit the key and it will leave the defaults unchanged. (You can try out `_.merge` in the browser console if you want to see the exact behavior)